index.js

```diff
@@ -0,0 +1,95 @@
+const playerImg = document.querySelector(".player img")
+const playerName = document.querySelector(".player h2")
+const playerNick = document.querySelector(".player em")
+const playerLikes = document.querySelector(".likes")
+const playerGoalsUl = document.querySelector("#goals")
+const likeBtn = document.querySelector(".like-button")
+const newGoalForm = document.querySelector("#new-goal-form")
+
+const handleNewGoalFormSubmit = event => {
+	event.preventDefault()
+	const link = event.target.link.value
+	const description = event.target.description.value
+	const newGoal = {
+		playerId: playerName.dataset.id,
+		link: link,
+		description: description
+	}
+	playerNewGoalFetchPost(newGoal)
+	event.target.reset
+}
+
+likeBtn.addEventListener("click", () => playerLikeFetchPatch())
+newGoalForm.addEventListener("submit", handleNewGoalFormSubmit)
+
+const playerLikeFetchPatch = () => {
+	return fetch("http://localhost:3000/players/1", {
+		method: 'PATCH',
+		headers: {'Content-Type': 'application/json'},
+		body: JSON.stringify({likes: parseInt(playerLikes.textContent) + 1})
+	})
+		.then(response => response.json())
+		.then(updatedPlayer => {
+			console.log('Success:', updatedPlayer)
+			playerLikes.textContent = updatedPlayer.likes
+		})
+}
+
+const playerNewGoalFetchPost = (newGoal) => {
+	return fetch("http://localhost:3000/players/1/goals", {
+		method: 'POST',
+		headers: {'Content-Type': 'application/json'},
+		body: JSON.stringify({
+			playerId: newGoal.playerId,
+			link: newGoal.link,
+			description: newGoal.description
+		})
```

Comment:
Since the `newGoal` object already has the correct keys for this request, this isn't necessary - you can just do:

```js
body: JSON.stringify(newGoal)
```